tools/mo/openvino/tools/mo/utils/cli_parser.py

```diff
@@ -709,25 +709,6 @@ def transform_param_to_str(value):
 }
 
 
-class DeprecatedStoreTrue(argparse.Action):
-    def __init__(self, nargs=0, **kw):
-        super().__init__(nargs=nargs, **kw)
-
-    def __call__(self, parser, namespace, values, option_string=None):
-        dep_msg = "Use of deprecated cli option {} detected. Option use in the following releases will be fatal. ".format(option_string)
-        if 'fusing' in option_string:
-            dep_msg += 'Please use --finegrain_fusing cli option instead'
-        log.error(dep_msg, extra={'is_warning': True})
-        setattr(namespace, self.dest, True)
-
-
-class DeprecatedOptionCommon(argparse.Action):
-    def __call__(self, parser, args, values, option_string):
-       dep_msg = "Use of deprecated cli option {} detected. Option use in the following releases will be fatal. ".format(option_string)
-       log.error(dep_msg, extra={'is_warning': True})
-       setattr(args, self.dest, values)
-
-
```

Comment:
Please do not remove if because we can deprecate future options. Just remove mention of `--finegrain_fusing`, etc.

Reply:
done